app/src/utils/api.ts

```diff
@@ -188,9 +188,12 @@ export const RepoAPICall = async (
       const data = await response.json();
       return data;
     } else {
+      console.log(`Response from Repo API ${apiUrl} was not a 200`);
       return undefined;
     }
   } catch (error) {
+    console.log(`error fetching Repo API ${apiUrl}`);
```

Comment:
You can remove the logs. I think most of this is updated in the error handling PR.